daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/StablePackageTest.scala

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+package archive
+
+import java.io.File
+import com.daml.bazeltools.BazelRunfiles
+import com.daml.lf.language.StablePackage
+import org.scalatest._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class StablePackageTest
+    extends AnyWordSpec
+    with Matchers
+    with Inside
+    with BazelRunfiles
+    with Inspectors
+    with TryValues {
+
+  private def resource(path: String): File = {
+    val f = new File(path).getAbsoluteFile
+    require(f.exists, s"File does not exist: $f")
+    f
+  }
+
+  "stable packages" should {
+
+    lazy val darFile = resource(rlocation("daml-lf/archive/DarReaderTest.dar"))
+    lazy val pkgsInDar = UniversalArchiveDecoder.assertReadFile(darFile).all.toMap
+
+    // fix as new packages are added to the std lib
+    assert(pkgsInDar.size == StablePackage.values.size + 3)
+
+    "be in any dar file" in {
+      val pkgIdsInDar = pkgsInDar.keySet
+      StablePackage.values.foreach(pkg => pkgIdsInDar should contain(pkg.packageId))
```

Comment:
only true until we add stable packages for new LF versions or for 1.dev; this test should also check that stable packages with a version greater than our dar's should _not_ be present.